utils/src/lib.rs

```diff
@@ -1011,6 +1018,67 @@ pub fn init_tracing_subscriber() {
         .ok();
 }
 
+pub async fn compile_contract_source(
```

Comment:
@onbjerg thinking about this a bit. Problem with having this in the `cli` crate is that we introduce a cyclical dependency between the `evm` crate and the `cli` crate (evm references itself)

One level higher, maybe the cli crate isn't the right place for the compile module at all, it could be in `common` instead? 



wyt?

Reply:
I think OK to pull it in common